checklists/checklist.php

```diff
@@ -470,7 +470,7 @@ function changeImageSource(elem){
 							if(array_key_exists('vern',$sppArr)){
 								echo ' - <span class="vern-span">'.$sppArr['vern'] . '</span>';
 							}
-							if($clid && $clArray['dynamicsql']){
+							if($clid && $clArray['dynamicsql'] && $SYMB_UID){
```

Comment:
Wondering if limiting access to signed in users needs to be configurable on a per portal basis?